static/scss/program-page.scss

```diff
@@ -308,7 +308,7 @@
 
   }
 
-  .block-course_table {
+  .block-course_table, .block-benefits_table {
```

Comment:
This table design will exactly be the same, But the reason for creating this class with the name that when the table block renders it looks for a design class that matches the generated template name. e.g. `.block-course_table` was created to provide style for table block named `Course table` in CMS. Similarly, we added a table block with the name `Benefits table` so we had to provide a class matching the named syntax.

We can either go with this implementation or we can name both the table blocks as `Table` in the Program Tab(Currently named `Course table`) page and benefits page(Currently named `Benefits table`) and we would be able to use the same design class for both.

You can reference this with PR's first attached image. Let me know what you suggest.

Reply:
Ok, I thought that might be the case. 

I'd like to get rid of that "Benefits" in the template name, since we'll be using this template for many pages, not just "Benefits". But that's out of scope for this PR.